AirCasting/SearchAndFollow/MapSessionsDownloader/APIStruct/MapDownloaderMeasurementType.swift

```diff
@@ -16,6 +16,13 @@ extension MapDownloaderMeasurementType {
         }
     }
     
+    var apiNameSufix: String {
```

Comment:
typo: `Suffix`. It should probably also be renamed to something better... How about `sensorNameSuffix` (see also the prefix name change proposition)